src/routes/(console)/project-[region]-[project]/databases/database-[database]/table-[table]/export/+page.svelte

```diff
@@ -0,0 +1,320 @@
+                    delimiter: delimiterMap[delimiter],
+                    header: includeHeader,
+                    notify: emailOnComplete
+                });
+
+            addNotification({
+                type: 'success',
+                message: 'CSV export has started'
+            });
+
+            trackEvent(Submit.DatabaseExportCsv);
+
+            await goto(tableUrl);
+        } catch (error) {
+            addNotification({
+                type: 'error',
+                message: error.message
+            });
+
+            trackError(error, Submit.DatabaseExportCsv);
+        }
+    }
+
+    onMount(() => {
+        loadBuckets();
+        initializeColumns();
+        localQueries = new Map($queries);
+    });
+</script>
+
+<Wizard title="Export CSV" columnSize="s" href={tableUrl} bind:showExitModal confirmExit column>
+    <Form bind:this={formComponent} onSubmit={handleExport}>
+        <Layout.Stack gap="xxl">
+            <Fieldset legend="Destination">
+                <Layout.Stack gap="l">
+                    {#if loadingBuckets}
+                        <Layout.Stack gap="xs">
+                            <Typography.Text as="label" for="bucket" variant="m-500"
+                                >Bucket</Typography.Text>
+                            <Skeleton variant="line" width="100%" height={35} />
+                        </Layout.Stack>
+                    {:else if buckets && buckets.buckets.length > 0}
+                        <InputSelect
+                            id="bucket"
+                            label="Bucket"
+                            required
+                            bind:value={selectedBucket}
+                            placeholder="Select bucket"
+                            options={buckets.buckets.map((bucket) => ({
+                                value: bucket.$id,
+                                label: bucket.name
+                            }))} />
+                    {:else}
+                        <Typography.Text variant="m-400" color="--fgcolor-neutral-tertiary">
+                            No storage buckets found. Please create a bucket first.
+                        </Typography.Text>
+                    {/if}
+
+                    <InputText id="filename" label="Filename" bind:value={filename} />
+                </Layout.Stack>
+            </Fieldset>
+
+            <Fieldset legend="Columns">
+                <Layout.Stack gap="l">
+                    <Layout.Stack direction="row" gap="s" alignItems="center">
+                        <Button compact on:click={selectAllColumns}>Select all</Button>
+                        <span style:height="20px">
+                            <Divider vertical />
+                        </span>
+                        <Button compact on:click={deselectAllColumns}>Deselect all</Button>
+                    </Layout.Stack>
+
+                    <Layout.Grid columns={3} gap="l">
```

Comment:
we might have to look for small screens here, not sure if 3 would fit nicely in case of slightly longer column names. The component supports more sizes via - `columns`, `columnsS|XS|XXS`, `columnsL|XL|XXL`. Please check with design once for other viewports 👍